### PR TITLE
DSG-2691: accept invitation followup

### PR DESCRIPTION
**Pull request by davidyoung77**
# Basics

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Styling and layout changes
- [ ] Code style changes (formatting, local variables)
- [ ] Refactoring (no functional changes)
- [ ] Build or CI related changes
- [ ] Tests
- [ ] Other... Please describe:

## What changes does this PR introduce?

- changes to the notification
- responsive styles
- improvements the the expiration imminent handling

## Screenshot of the affected changes

Screenshot 2024-02-02 at 10 48 36 AM
Screenshot 2024-02-02 at 12 47 57 PM
Screenshot 2024-02-02 at 12 47 35 PM
Screenshot 2024-02-05 at 10 59 20 AM

## Checklist

- [ ] I have updated the documentation accordingly
- [ ] I have confirmed all acceptance criteria defined by the story have been met
- [ ] I have confirmed that Sonar Qube findings for new code are reviewed and fixed, or will be whitelisted

### CODE REVIEW

File: libs/employer/feature/dashboard/src/lib/components/account-access/claim-invitation/claim-invitation.component.html [hunk 1]
```diff
@@ -32,4 +36,9 @@ <h1>Invitations</h1>
   >
 </div>
 
+<div class="error-message-wrapper" *ngIf="(errorMessage$ | async) as _errorMessage">
```
Comment (thechechogomez) [inline libs/employer/feature/dashboard/src/lib/components/account-access/claim-invitation/claim-invitation.component.html:39]: Seems `_errorMessage` is not used, would we need it for something in the future?

File: libs/employer/feature/dashboard/src/lib/components/account-access/claim-invitation/claim-invitation.component.ts [hunk 2]
```diff
@@ -11,26 +11,30 @@ import {
...
   standalone: true,
   styleUrls: ['./claim-invitation.component.scss'],
   templateUrl: './claim-invitation.component.html',
 })
 export class ClaimInvitationComponent {
   public expirationStatus = InvitationExpirationStatus;
-
   public details$;
-
+  public errorMessage$: Observable<ErrorMessage>;
```
Comment (thechechogomez) [inline libs/employer/feature/dashboard/src/lib/components/account-access/claim-invitation/claim-invitation.component.ts:37]: Wouldn't it be better to have a primitive type such as `boolean` instead on the enum? Seems we are not using the enum for anything other that to type the error.

**Review by thechechogomez — APPROVED** [review]
Look good, have a few comments.

File: libs/employer/feature/dashboard/src/lib/components/account-access/claim-invitation/claim-invitation.component.html (hunk 1 above)
Comment (davidyoung77) [reply libs/employer/feature/dashboard/src/lib/components/account-access/claim-invitation/claim-invitation.component.html:39]: I was thinking potentially in the future, but I will change to boolean

**Review by thechechogomez — APPROVED** [review]

**Review by thechechogomez — APPROVED** [review]

**Review by thechechogomez — APPROVED** [review]

**Pull request merged**